public/css/admin_profile.css

```diff
@@ -116,19 +116,10 @@ aside h1 {
     padding: 20px;
 }
 
-/* The Close Button */
```

Comment:
Is there any reason you removed all the styling from the admin page? The create user and edit user modals are now blank. Also, the delete user button has disappeared and is unclickable now. This comment applies to the whole file.

Reply:
Recovered and adopted the similar style to searchbar.